lib/hbs-helpers.js

```diff
@@ -9,6 +9,15 @@ module.exports.register = function (handlebars) {
 		return a ? a : b;
 	});
 
+	/**
+	 * creates image src for generated assets
+	 */
+	handlebars.registerHelper('img', function (width, height, src) {
```

Comment:
the name of this helper should be adjusted too with bioHelper prefix

Reply:
I removed this helper from this pull and put it in another pr.